.github/workflows/publish.yaml

```diff
@@ -0,0 +1,22 @@
+name: "Publish NPM package"
+
+on:
+  pull_request:
+    branches: 
+      - main
+    types: [closed]
+
+jobs:
+  if_merged:
+    if: startsWith(github.event.pull_request.title, 'Bump Package Version') && github.event.pull_request.merged == true 
```

Comment:
@madhav-devslane I am approving this for now but for future let's look at some standard solutions to use tags for releases